README.md

```diff
@@ -4,7 +4,52 @@ inspired by
 
 https://github.com/FredrikOseberg/react-chatbot-kit
 
----
+### Make your own configuration
+
+#### Questions`s roadmap
+
+First of all you need to make your own question`s roadmap dependencies.
+
+You need to create your `config.js` file.
+
+See example of `config.js` in directory `examples`.
+
+`config.js` file have four fields:
+
+1) initMessages - Array of messages that will see users when open a chat bot.
+2) initQuestions. It`s a start questions of roadmap for users. Array consist of questions names.
```

Comment:
2) `initQuestions`. Here are stored initial roadmap questions for users. The array consists of question names